src/NaiveBayes.jl

```diff
@@ -5,13 +5,16 @@ using KernelDensity
 using Grid
 
 export NBModel,
-       MultinomialNB,
-       GaussianNB,
-       KernelNB,
-       fit,
-       predict,
-       predict_proba,
-       predict_logprobs
+        MultinomialNB,
+        GaussianNB,
+        KernelNB,
+        HybridNB,
+        fit,
+        predict,
+        predict_proba,
```

Comment:
Should we really expose all these methods to the user? I think we should export only fit and predict. All the others are internal